File: problems/94/Main.cpp
#include <iostream>
#include <cstdio>
#include <algorithm>
#include <cstring>
#include <string>
#include <cctype>
#include <stack>
#include <queue>
#include <list>
#include <vector>
#include <map>
#include <sstream>
#include <cmath>
#include <bitset>
#include <utility>
#include <set>
#include <numeric>
#include <ctime>
#include <math.h>

using namespace std;

class Solution {
	public:
		vector<int> getRow(int A);
};

//--SPLIT--

//-vector<int> Solution::getRow(int A) {
//-}

//--SPLIT--
vector<int> Solution::getRow(int A) {
	vector<int> result(A + 1);
	return result;
}

//--SPLIT--
int main() {
	int testcases;
	cin >> testcases;
	for (int t = 0; t < testcases; t++) {
		int A; cin >> A;
		Solution solution;
		vector<int> result = solution.getRow(A);
		for (int i = 0; i < result.size() ;i++) {
			cout << result[i] << " ";
		}
		cout << "\n";
	}
	return 0;
}

File: problems/94/Main.py
from __future__ import print_function
import math

#--SPLIT--
#-class Solution:
	# @param A : integer
	# @return a list of integers
#-	def getRow(self, A):


#--SPLIT--
class Solution:
	def getRow(self, A):
		return []

#--SPLIT--

if __name__ == "__main__":
    test_cases = int(raw_input())
    for i in range(test_cases):
        A = int(raw_input())
        solution = Solution()
        result = solution.getRow(A) 
        for r in result:
            print(str(r) + " ", end="")
        print("")